packages/next/server/config-schema.ts

```diff
@@ -563,7 +563,15 @@ const configSchema = {
       type: 'object',
     },
     optimizeFonts: {
-      type: 'boolean',
+      properties: {
```

Comment:
I think we still need to accept the boolean value here to prevent breaking existing configs.